public/app/features/trails/Integrations/DataTrailEmbedded.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { AdHocVariableFilter } from '@grafana/data';
+import { SceneComponentProps, SceneObjectBase, SceneObjectState, SceneTimeRangeLike } from '@grafana/scenes';
+
+import { DataTrail } from '../DataTrail';
+import { getDataTrailsApp } from '../DataTrailsApp';
+import { OpenEmbeddedTrailEvent } from '../shared';
+
+export interface DataTrailEmbeddedState extends SceneObjectState {
+  timeRange: SceneTimeRangeLike;
+  metric?: string;
+  filters?: AdHocVariableFilter[];
+  dataSourceUid?: string;
+}
+export class DataTrailEmbedded extends SceneObjectBase<DataTrailEmbeddedState> {
+  static Component = DataTrailEmbeddedRenderer;
+
+  public trail: DataTrail;
+
+  constructor(state: DataTrailEmbeddedState) {
+    super(state);
+
+    this.trail = buildDataTrailFromState(state);
+    this.trail.addActivationHandler(() => {
+      this.trail.subscribeToEvent(OpenEmbeddedTrailEvent, this.onOpenTrail);
+    });
+  }
+
+  onOpenTrail = () => {
+    getDataTrailsApp().goToUrlForTrail(this.trail.clone({ embedded: false }));
+  };
```

Comment:
I think we can remove this bit, not sure why I did it with this event to be honest. 

If we change the open button to this (in MetricScene). It's a normal link, and user can decide if they want to open it in new tab or not by CMD/CTRL+click. 

```
 {trail.state.embedded && (
              <LinkButton href={getUrlForTrail(trail)} variant={'secondary'}>
                Open
              </LinkButton>
            )}
```

Reply:
Is this what you had in mind?
- 611c211